src/seeds.organization.cpp

```diff
@@ -154,10 +212,11 @@ ACTION organization::create(name sponsor, name orgaccount, string orgfullname, s
         norg.org_name = orgaccount;
         norg.owner = sponsor;
         norg.planted = quantity;
-        norg.status = 0;
+        norg.status = regular_org;
```

Comment:
@IanMendozaJaimes did you change status to be a name instead of int? 

Problem is we then need to migrate table data, wipe table, and migrate back